ban/tests/test_models.py

```diff
@@ -349,6 +349,18 @@ def test_housenumber_as_relation():
     }
 
 
+def test_housenumber_municipality_is_cached(mocker, sql_spy):
+    HouseNumberFactory()
+    # Reload the instance to clear property cache from Diff creation.
+    housenumber = models.HouseNumber.first()
+    expected = housenumber.parent.municipality
+    sql_spy.reset_mock()
+    assert housenumber.municipality == expected
+    assert sql_spy.call_count == 1
+    assert housenumber.municipality == expected
+    assert sql_spy.call_count == 1
```

Comment:
I guess that's because `sql_spy` is here to verify that we do not issue a subsequent SQL call when we retrieve that property again. I guess that `cached_property` is well tested elsewhere but why not :)